dspace-api/src/main/java/org/dspace/statistics/SolrLogger.java

```diff
@@ -706,13 +712,21 @@ public void execute(String query) throws SolrServerException, IOException {
             process(response.getResults());
 
             // Run over the rest
-            for (int i = 10; i < numbFound; i += 10)
+            for (int i = maxDocumetsPerProcess; i < numbFound; i += maxDocumetsPerProcess)
             {
-                params.put("start", String.valueOf(i));
+                if(commitAtEveryProcess) {
+                    commit();
```

Comment:
arriba del commit() le pondría un comentario tipo
//force solr commit to avoid trigering autocommits inside solr